docs/openstack-openedx/production-deployment.md

```diff
@@ -1,7 +1,8 @@
 # OpenStack Production Deployment (Proposal)
 
 edX does not officially provide any supported method for deploying a production-grade instance of Open edX. However, the following *proposed* method for setting up a production-grade instance on OpenStack is available to the community, and recommended by OpenCraft. For context, this method of deployment is used in production for over 100 production instances of Open edX on [OpenCraft's hosting platform](http://opencraft.com/hosting/).
- 
+
+To deploy to Amazon Web Services, see [AWS deployment](../aws-openedx/aws-deployment.md).
```

Comment:
Please remove this statement -- this document is specifically about "supported" production deployments, which are currently only OpenStack.